docs/sphinx/api/rest/.openapi.yaml

```diff
@@ -87,6 +87,11 @@ definitions:
             type: string
         example: ["inline", "canonicalize", "quake-to-qir"]
         description: List of IR passes to be applied on the server.
+      serializedCodeExecutionContext:
```

Comment:
These new fields are not supported on NVQC right now; I'm wondering if we should hold off on documenting them here until we deploy them to NVQC.